can_tools/scrapers/official/federal/CDC/cdc_community_level.py

```diff
@@ -0,0 +1,83 @@
+import pandas as pd
+from can_tools.scrapers import CMU
+from can_tools.scrapers.official.base import FederalDashboard
+
+
+COMMUNITY_LEVEL_MAP = {"Low": 0, "Medium": 1, "High": 2}
+
+
+class CDCCommunityLevel(FederalDashboard):
+    """Scraper to collect CDC County Community Level.
+
+    Data is updated weekly on Thursdays, and is only available at the county level.
+    There is no historical data from before February, 2022, and there does not appear to be a
+    secondary dataset containing the historical data (as there is with the testing datasets).
+    """
+
+    has_location = True
+    location_type = "county"
+    source = "https://data.cdc.gov/Public-Health-Surveillance/United-States-COVID-19-Community-Levels-by-County-/3nnm-4jni"
+    source_name = "Centers for Disease Control and Prevention"
+    provider = "cdc_community_level"
+    csv_url = "https://data.cdc.gov/api/views/3nnm-4jni/rows.csv?accessType=DOWNLOAD"
+
+    variables = {
```

Comment:
variable descriptions/units can be found under "Columns in this Dataset" here: https://data.cdc.gov/Public-Health-Surveillance/United-States-COVID-19-Community-Levels-by-County-/3nnm-4jni